modules/btcd/module.cpp

```diff
@@ -40,8 +40,10 @@ namespace bitcoinfuzz
             auto pointer{BTCDDesBlock(script_data)};
             std::string result(pointer);
             BTCDFreeString(pointer);
-            std::vector<bool> final_result{"true" == result};
-            return final_result;
+            if (result.empty() || result == "0") {
+                return std::nullopt;
+            }
+            return result;
```

Comment:
```suggestion
            return result;
```
We can return the result directly here